src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/models/ManagedIndexConfig.kt

```diff
@@ -69,28 +69,27 @@ data class ManagedIndexConfig(
     override fun toXContent(builder: XContentBuilder, params: ToXContent.Params): XContentBuilder {
         builder
             .startObject()
-                .startObject(MANAGED_INDEX_TYPE)
-                    .field(NAME_FIELD, jobName)
-                    .field(ENABLED_FIELD, enabled)
-                    .field(INDEX_FIELD, index)
-                    .field(INDEX_UUID_FIELD, indexUuid)
-                    .field(SCHEDULE_FIELD, jobSchedule)
-                    .optionalTimeField(LAST_UPDATED_TIME_FIELD, jobLastUpdatedTime)
-                    .optionalTimeField(ENABLED_TIME_FIELD, jobEnabledTime)
-                    .field(POLICY_NAME_FIELD, policyName)
-                    .field(POLICY_SEQ_NO_FIELD, policySeqNo)
-                    .field(POLICY_PRIMARY_TERM_FIELD, policyPrimaryTerm)
-                    .field(POLICY_FIELD, policy, XCONTENT_WITHOUT_TYPE)
-                    .field(CHANGE_POLICY_FIELD, changePolicy)
-                .endObject()
+            .startObject(MANAGED_INDEX_TYPE)
```

Comment:
NP: I don't have a strong preference to the format of the indentation for ```toXContent()``` but the other models seems to write it the old way so let's decide on one style to keep it consistent.